roles/grid/tasks/voms.yml

```diff
@@ -15,13 +15,33 @@
     mode: 0755
   loop: "{{ grid_vos }}"
 
+- name: Install VOMS LSC files
+  package:
+    state: installed
+    name: "wlcg-voms-{{ vo }}"
+  loop: "{{ grid_vos }}"
+  loop_control:
+    loop_var: vo
+  when: vo in ['alice', 'atlas', 'cms', 'lhcb', 'ops', 'dteam']
```

Comment:
do we need lhcb? I think it can go.

Reply:
we don't use it but it's only in the set of available wlcg-voms-{{vo}} rpms and for completness the task should include the entire set in case anybody in the future wants to use the role and needs lhcb configured

Reply:
ah, I see now. this is the list of VOs for which packages exist, it's not part of the actual run (that's grid_vos as the loop var)